package.json

```diff
@@ -19,11 +19,14 @@
     }
   },
   "jest": {
-    "coveragePathIgnorePatterns": ["<rootDir>/src/lib/static/"],
+    "coveragePathIgnorePatterns": [
+      "<rootDir>/src/lib/static/"
+    ],
     "moduleNameMapper": {
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
       "\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
-    }
+    },
+    "setupFiles": ["<rootDir>/src/config/jestSetup.js"]
```

Comment:
Should this be split into multiple lines like `coveragePathIgnorePatterns`? Does the Airbnb Style Guide say anything on the topic of newlines in array and object declaration?

Reply:
I would think that JS style guides should apply 1:1 to JSON, given that it's functional JavaScript after all. But I'm still a layperson on the matter. No problem either way.